package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "acorn",
-  "version": "v0.10.0-rc2",
+  "version": "v0.10.0-rc3",
   "lockfileVersion": 3,
   "requires": true,
   "packages": {
     "": {
       "name": "acorn",
-      "version": "v0.10.0-rc2",
+      "version": "v0.10.0-rc3",
```

Comment:
Not sure why these weren't already updated, since I didn't make any changes to `package.json`. 🤷‍♂️

Reply:
My fault.

Reply:
Yeah we should enforce that at ci level, like doing a npm install and check if git is dirty